<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <widget class="QWidget" name="gameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>780</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/assets/Images/569.bmp);
border:5 solid black;
border-left:1;
border-right:1</string>
    </property>
    <widget class="QTableWidget" name="playerDataWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>231</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:5px solid rgb(204,102,0);
background-image:url(:/assets/Images/221.bmp);
color:white;
text-align:center</string>
     </property>
    </widget>
    <widget class="QListWidget" name="levelDataWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>570</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:5px solid rgb(204,102,0);
border-top:0;background-image:url(:/assets/Images/221.bmp);
color:white;</string>
     </property>
    </widget>
    <widget class="MagicMap" name="mapWidget" native="true">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>680</width>
       <height>680</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:3px solid rgb(204,102,0);
background-image:url(:/assets/Images/221.bmp);</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="keysDataWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>380</y>
       <width>231</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:5px solid rgb(204,102,0);background-image:url(:/assets/Images/221.bmp);
color:white;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="optionsDataWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>650</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:5px solid rgb(204,102,0);background-image:url(:/assets/Images/221.bmp);
color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="gainLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>1011</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
border:3px solid rgb(204,102,0);
background-image:url(:/assets/Images/221.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="successLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>1011</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>70</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 74, 79);
background:transparent;
border:0;
text-shadow:5px 2px 6px white;</string>
     </property>
     <property name="text">
      <string>战斗胜利</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="MagicMenu" name="MenuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>780</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black</string>
    </property>
   </widget>
   <zorder>MenuWidget</zorder>
   <zorder>gameWidget</zorder>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Open Map</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Save Record</string>
   </property>
  </action>
  <action name="actionLoad_Record">
   <property name="text">
    <string>Load Record</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagicMap</class>
   <extends>QWidget</extends>
   <header>magicmap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MagicMenu</class>
   <extends>QWidget</extends>
   <header location="global">magicmenu.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
